src/instruments/src/EFB/Localization/en.json

```diff
@@ -499,6 +499,7 @@
       "PauseAtTod": "T/D Pause",
       "PauseAtTodDistance": "T/D Pause Distance (nm)",
       "SeparateTillerFromRudderInputs": "Separate Tiller from Rudder Inputs",
+      "SyncEfis": "Sync EFIS controls from Captain to FO",
```

Comment:
There should not be a diff in the localisation files here, this is handled through localazy

If a new key needs to be added that would be done through there.

Reply:
No, this is ok. I asked that he runs the download of language files locally so that the PR has a commit of all files with the new key. By doing so it would also get all the latest translations which is ok. 
Normally these would come into the repo by the regular github action anyway. 
The build overwrites these with the latest files so that there is no problem.